contracts/dapis/DapiServer.sol

```diff
@@ -208,7 +247,50 @@ contract DapiServer is
             this.fulfillRrpBeaconUpdate.selector
         );
         requestIdToBeaconId[requestId] = beaconId;
-        emit RequestedRrpBeaconUpdateRelayed(
+        emit RequestedRelayedRrpBeaconUpdate(
+            beaconId,
+            sponsor,
+            msg.sender,
+            requestId,
+            airnode,
+            relayer,
+            templateId
+        );
+    }
+
+    /// @notice Creates an RRP request for the Beacon to be updated by the relayer
+    /// @param airnode Airnode address
+    /// @param endpointId Endpoint ID
+    /// @param parameters Parameters
+    /// @param relayer Relayer address
+    /// @param sponsor Sponsor address
+    /// @return requestId Request ID
+    function requestRelayedRrpBeaconUpdateWithEndpoint(
```

Comment:
I feel like this function is just bloat

Reply:
Worst case is it's for the sake of completeness, but I can actually see it being a thing in the future